### PR TITLE
Fix GPU memory consumption with retina tiles and browser zoom

### PR DESCRIPTION
**Pull request by davojta**
1) switch off will-change
2) add no3d option for layer options
3) add test page

Discussed in https://github.com/Leaflet/Leaflet/issues/4578

### CODE REVIEW

**Comment by mourner** [comment]
There's a reason the was `will-change` in CSS — as far as I remember, `flyTo` animations were excruciatingly slow without it on certain browsers/devices, so removing it will inevitably regress performance in some cases. We may revisit this now but test very thoroughly on iOS, Android, all desktop browsers etc.

**Comment by davojta** [comment]
@mourner I think we can do it with special flag or just mention it in docs 
They main benefit is option to switch off translate3d for retina tiles and fallback to translate implementation without flooding GPU memory. There is a rare edge case anyway (browser zoom + retina tiles but cause browser crash )

**Comment by kaynz** [comment]
Any news on this,  @mourner ? :)

File: src/dom/DomUtil.js [hunk 1]
```diff
@@ -184,11 +184,11 @@ L.DomUtil = {
 	// Resets the 3D CSS transform of `el` so it is translated by `offset` pixels
```
Comment (Falke-Design) [inline src/dom/DomUtil.js:184]: Docs: parameter `no3d` missing

File: src/dom/DomUtil.js [hunk 2]
```diff
@@ -198,14 +198,13 @@ L.DomUtil = {
 	// Sets the position of `el` to coordinates specified by `position`,
```
Comment (Falke-Design) [inline src/dom/DomUtil.js:198]: Docs: parameter `no3d` missing

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
A part of this has already landed in #7872. Are the changes of this PR still relevant?

**Pull request closed without merging**